InstagramKit/Engine/InstagramEngine.h

```diff
@@ -562,31 +528,26 @@ NS_ASSUME_NONNULL_BEGIN
                         withSuccess:(InstagramResponseBlock)success
                             failure:(nullable InstagramFailureBlock)failure;
 
-
 /**
- *  Get the list of users this user follows.
+ *  Get the list of users this self follows.
```

Comment:
This statement is grammatically incorrect. Please use the term `the logged in user` in place of `this self` in all documentation comments.